modules/core/src/components/log-viewer/core-3d-viewer.js

```diff
@@ -155,13 +155,13 @@ export default class Core3DViewer extends PureComponent {
   _onMetrics = deckMetrics => {
     if (this.props.debug) {
       const metrics = {
-        fps: deckMetrics.fps,
-        redraw: deckMetrics.redraw || 0
+        fps: deckMetrics.frameRate.hz,
```

Comment:
is `fps` the same as `hz`?

Reply:
No, all stats have a field called `hz`, and `fps` refers to how many animation frame are requested per second.

This is experimental API and we will need to do a thorough audit in 7.1 time frame.

Reply:
Why not `frameRate.count like` the other metrics?  Also `hz` gives a float with many decimals. 

<img width="460" alt="Screen Shot 2019-05-06 at 4 41 22 PM" src="https://user-images.githubusercontent.com/2488251/57261717-d9af0800-701d-11e9-97de-6ae6db30ba34.png">